srcjs/init_shiny.js

```diff
@@ -207,6 +207,15 @@ function initShiny() {
   }
   exports.initializeInputs = initializeInputs;
 
+  function getIdFromEl(el) {
+    var $el = $(el);
+    var bindingAdapter = $el.data("shiny-output-binding");
+    if (!bindingAdapter)
+      return null;
+    else
+      return bindingAdapter.getId();
```

Comment:
This PR looks good to me, but I guess here you meant `bindingAdapter.getId(el)`? Other than that, I think it should be ready to go after grunt/git commit -a/git rebase/force push.